tools/clang/test/CodeGenSPIRV/select.long.lit.hlsl

```diff
@@ -0,0 +1,29 @@
+// RUN: %dxc -T ps_6_0 -HV 2018 -E main
+
+// Check that the literals get a 64-bit type, and the result of the select is
+// then cast to an unsigned 64-bit value.
+void foo(uint x) {
```

Comment:
nit: maybe add the same tests for the enable-16bit-types flag/types?

Reply:
done.